packages/netlify/src/ssr-function.ts

```diff
@@ -21,6 +21,7 @@ export const createExports = (manifest: SSRManifest, _args: Args) => {
 			let locals: Record<string, unknown> = {};
 
 			if (request.headers.has('x-astro-locals')) {
+				// biome-ignore lint/style/noNonNullAssertion: safe because we checked before
```

Comment:
I would love if typescript would be able to handle those checks on Maps....

Reply:
I can change the code. The `.get` method returns undefined, so we can check the result. We wouldn't need the bang operator

Reply:
Done in [`0b9fc0e` (#111)](https://github.com/withastro/adapters/pull/111/commits/0b9fc0e2b297dddd50e613519061b530d86cbead)

Reply:
Oh yeah that would work, I think rewriting the code makes sense.

I just think Typescript should support Map's `.has()`